### PR TITLE
Releasing HUB-1333 and HUB-1347

### PR DESCRIPTION
**Pull request by mishraa8**

### CODE REVIEW

**Comment by lkemmerer** [comment]
Is the commit history going to get weirder and weirder because we didn't do a fast-forward merge of develop into master last time? I'd like us to figure out how to get things into a state that master's history is the same as develop up to new work that we want to merge in. Maybe to get there we would need to do a one-time reset of `develop` to what's on `master`? I'm not sure.

**Comment by lkemmerer** [comment]
To help this from happening in the future, I updated the master branch to require linear history -- https://github.com/flightstats/hub/settings/branch_protection_rules/2131062 I guess we can see how merging this into master goes based on that restriction?

File: src/main/java/com/flightstats/hub/channel/ChannelResource.java [hunk 1]
```diff
@@ -214,18 +214,6 @@ public Response insertValue(@PathParam("channel") String channelName,
         }
     }
 
-    @POST
-    @Consumes("multipart/*")
-    @Produces(MediaType.APPLICATION_JSON)
-    @Path("/batch")
-    public Response insertBatch(@PathParam("channel") final String channelName,
```
Comment (lkemmerer) [inline src/main/java/com/flightstats/hub/channel/ChannelResource.java:221]: These changes should already be in master, no?

**Comment by mishraa8** [comment]
Yes, Agree with your point @lkemmerer , master and develop branch history would never be same, and it can go worse.
We can do deleting Develop branch and create Develop branch again. Hope that will make it clear. Going forward with Linear History enabled it will be clean and same as master history.

**Pull request closed without merging**